### ISSUE TITLE
MAME2010 fails to build

### ISSUE THREAD

**Issue by pdrome** [libretro/mame2010-libretro#6]
MAME2010 fails to build using buildbot.

**Comment by pdrome** [comment]
Looking into this, it might be related to src/osd/retro/libco/armeabi_asm.S which was deleted last June still being used in the OSD library.  See here:

src/osd/retro/retro.mak

**Comment by r-type** [comment]
Please povied an pastbin of the error.

**Comment by pdrome** [comment]
I will do so this evening when I get home from work.

**Comment by pdrome** [comment]
Sorry to take so long getting this up.  I think I've resolved the issue and I sent a pull request.  Here is the first pastebin:

http://pastebin.com/L6YugC1c

It appears as though the memory.h file in /src/emu/ and memory.h file in the toolchain were butting heads.  I renamed the file and it's link in /src/emu and it resolved this issue.  Here is the pastebin for the issue that followed:

http://pastebin.com/7afCmyM7

Here there was an old file /src/osd/retro/libco/armeabi_asm.S which had been deleted, but which was still referenced in /src/osd/retro/retro.mak.

I simply got rid of the reference in retro.mak and it compiled fine.  Just a not, you must use the j=1 compile option or there is a risk that certain dependencies won't be built by the time they are needed by other threads.

I don't know how to link the pull request, but I submitted it with a similar name.
